### PR TITLE
Link to the instructions on how to contribute to the N&N

### PR DESCRIPTION
**Pull request by fedejeanne**
Link to `CONTRIBUTING.md` from the main `README.md` and to the instructions from there.

The idea is to make the instructions more visible from GitHub.

### CODE REVIEW

**Comment by fedejeanne** [comment]
@jjohnstn would you like to review?

Additional question: is the part about Gerrit in `CONTRIBUTING.md` still up-to-date or should it be removed?

**Comment by akurtakov** [comment]
Gerrit is not used for anything by Eclipse TLP project and EF is looking to remove it by May 2025 (https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/wikis/Gerrit/Gerrit-and-Bugzilla-deprecation-and-migration-plan). So please remove the section.
P.S. Jeff is not Platform committer so he can't approve this patch.

**Comment by fedejeanne** [comment]
Thank you for the information @akurtakov , I removed the part about Gerrit.

I assume the rest is ok?

**Review by akurtakov — APPROVED** [review]

**Pull request merged**
